INFO 2023-04-17 15:11:13,912 autoreload run_with_reloader 36031 8486771520 Watching for file changes with StatReloader
INFO 2023-04-17 15:15:05,060 autoreload trigger_reload 36031 8486771520 /Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/pymysql/err.py changed, reloading.
INFO 2023-04-17 15:15:05,582 autoreload run_with_reloader 40088 8486771520 Watching for file changes with StatReloader
INFO 2023-04-17 15:15:14,222 autoreload run_with_reloader 40711 8486771520 Watching for file changes with StatReloader
INFO 2023-04-17 15:16:45,045 autoreload trigger_reload 40711 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/__init__.py changed, reloading.
INFO 2023-04-17 15:16:45,413 autoreload run_with_reloader 40745 8486771520 Watching for file changes with StatReloader
INFO 2023-04-17 15:17:10,027 autoreload run_with_reloader 40759 8486771520 Watching for file changes with StatReloader
INFO 2023-04-17 15:18:00,104 basehttp log_message 40759 6202847232 "GET / HTTP/1.1" 200 10731
WARNING 2023-04-17 15:18:00,152 basehttp log_message 40759 6202847232 "GET /static/admin/css/fonts.css HTTP/1.1" 404 1816
WARNING 2023-04-17 15:18:00,341 log log_response 40759 6219673600 Not Found: /favicon.ico
WARNING 2023-04-17 15:18:00,341 basehttp log_message 40759 6219673600 "GET /favicon.ico HTTP/1.1" 404 2113
INFO 2023-04-18 01:54:56,691 autoreload run_with_reloader 68863 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 01:55:12,206 autoreload trigger_reload 68863 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/settings/dev.py changed, reloading.
INFO 2023-04-18 01:55:12,429 autoreload run_with_reloader 68878 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 01:56:49,481 autoreload trigger_reload 68878 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/settings/dev.py changed, reloading.
INFO 2023-04-18 02:17:23,197 autoreload run_with_reloader 69273 8486771520 Watching for file changes with StatReloader
WARNING 2023-04-18 02:17:28,040 log log_response 69273 6195539968 Not Found: /
WARNING 2023-04-18 02:17:28,040 basehttp log_message 69273 6195539968 "GET / HTTP/1.1" 404 2167
WARNING 2023-04-18 02:17:28,277 log log_response 69273 6195539968 Not Found: /favicon.ico
WARNING 2023-04-18 02:17:28,277 basehttp log_message 69273 6195539968 "GET /favicon.ico HTTP/1.1" 404 2218
WARNING 2023-04-18 02:17:35,900 log log_response 69273 6195539968 Not Found: /home
WARNING 2023-04-18 02:17:35,901 basehttp log_message 69273 6195539968 "GET /home HTTP/1.1" 404 2197
INFO 2023-04-18 02:17:45,811 basehttp log_message 69273 6195539968 "GET /home/demo HTTP/1.1" 301 0
INFO 2023-04-18 02:17:45,837 basehttp log_message 69273 6212366336 "GET /home/demo/ HTTP/1.1" 200 5151
INFO 2023-04-18 02:17:45,875 basehttp log_message 69273 6229192704 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
INFO 2023-04-18 02:17:45,876 basehttp log_message 69273 6246019072 "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1152
INFO 2023-04-18 02:17:45,876 basehttp log_message 69273 6195539968 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
INFO 2023-04-18 02:17:45,877 basehttp log_message 69273 6279671808 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
INFO 2023-04-18 02:17:45,877 basehttp log_message 69273 6212366336 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121457
INFO 2023-04-18 02:17:45,877 basehttp log_message 69273 6262845440 "GET /static/rest_framework/js/jquery-3.5.1.min.js HTTP/1.1" 200 89476
INFO 2023-04-18 02:17:45,882 basehttp log_message 69273 6279671808 "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1268
INFO 2023-04-18 02:17:45,882 basehttp log_message 69273 6246019072 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1719
INFO 2023-04-18 02:17:45,882 basehttp log_message 69273 6195539968 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2023-04-18 02:17:45,883 basehttp log_message 69273 6229192704 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 39680
INFO 2023-04-18 02:17:45,899 basehttp log_message 69273 6229192704 "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
INFO 2023-04-18 02:20:10,749 autoreload trigger_reload 69273 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/urls.py changed, reloading.
INFO 2023-04-18 02:20:11,067 autoreload run_with_reloader 69326 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 02:25:02,129 autoreload trigger_reload 69326 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/settings/dev.py changed, reloading.
INFO 2023-04-18 02:25:02,522 autoreload run_with_reloader 71044 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 02:27:24,124 autoreload run_with_reloader 71099 8486771520 Watching for file changes with StatReloader
WARNING 2023-04-18 02:27:27,705 log log_response 71099 6130790400 Not Found: /
WARNING 2023-04-18 02:27:27,706 basehttp log_message 71099 6130790400 "GET / HTTP/1.1" 404 2285
INFO 2023-04-18 02:27:33,796 basehttp log_message 71099 6130790400 "GET /docs HTTP/1.1" 301 0
ERROR 2023-04-18 02:27:33,821 log log_response 71099 6147616768 Internal Server Error: /docs/
Traceback (most recent call last):
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/response.py", line 111, in render
    self.content = self.rendered_content
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework_swagger/renderers.py", line 54, in render
    self.set_context(data, renderer_context)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework_swagger/renderers.py", line 68, in set_context
    renderer_context['spec'] = OpenAPIRenderer().render(
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework_swagger/renderers.py", line 34, in render
    return OpenAPICodec().encode(data, **options)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework_swagger/renderers.py", line 16, in encode
    raise TypeError('Expected a `coreapi.Document` instance')
TypeError: Expected a `coreapi.Document` instance
ERROR 2023-04-18 02:27:33,821 basehttp log_message 71099 6147616768 "GET /docs/ HTTP/1.1" 500 98876
INFO 2023-04-18 02:30:05,612 autoreload run_with_reloader 71165 8486771520 Watching for file changes with StatReloader
ERROR 2023-04-18 02:30:12,659 log log_response 71165 6124859392 Internal Server Error: /docs/
Traceback (most recent call last):
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/defaulttags.py", line 1026, in find_library
    return parser.libraries[name]
KeyError: 'staticfiles'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/response.py", line 111, in render
    self.content = self.rendered_content
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework_swagger/renderers.py", line 55, in render
    return render(
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/defaulttags.py", line 1088, in load
    lib = find_library(parser, name)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/defaulttags.py", line 1028, in find_library
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'staticfiles' is not a registered tag library. Must be one of:
admin_list
admin_modify
admin_urls
cache
i18n
l10n
log
rest_framework
static
tz
ERROR 2023-04-18 02:30:12,660 basehttp log_message 71165 6124859392 "GET /docs/ HTTP/1.1" 500 195185
INFO 2023-04-18 02:36:34,146 autoreload run_with_reloader 71258 8486771520 Watching for file changes with StatReloader
ERROR 2023-04-18 02:36:48,969 log log_response 71258 6165803008 Internal Server Error: /docs/
Traceback (most recent call last):
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/template/response.py", line 111, in render
    self.content = self.rendered_content
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework_swagger/renderers.py", line 54, in render
    self.set_context(data, renderer_context)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework_swagger/renderers.py", line 68, in set_context
    renderer_context['spec'] = OpenAPIRenderer().render(
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework_swagger/renderers.py", line 34, in render
    return OpenAPICodec().encode(data, **options)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework_swagger/renderers.py", line 16, in encode
    raise TypeError('Expected a `coreapi.Document` instance')
TypeError: Expected a `coreapi.Document` instance
ERROR 2023-04-18 02:36:48,970 basehttp log_message 71258 6165803008 "GET /docs/ HTTP/1.1" 500 99013
INFO 2023-04-18 02:42:20,840 autoreload trigger_reload 71258 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/settings/dev.py changed, reloading.
INFO 2023-04-18 02:42:21,272 autoreload run_with_reloader 72205 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 02:43:06,019 autoreload trigger_reload 72205 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/urls.py changed, reloading.
INFO 2023-04-18 02:43:06,419 autoreload run_with_reloader 72226 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 02:44:34,161 autoreload trigger_reload 72226 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/urls.py changed, reloading.
INFO 2023-04-18 02:44:34,555 autoreload run_with_reloader 72248 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 02:45:20,090 autoreload run_with_reloader 72277 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 02:46:50,499 autoreload run_with_reloader 72312 8486771520 Watching for file changes with StatReloader
WARNING 2023-04-18 02:46:58,748 log log_response 72312 6190592000 Not Found: /docs/
WARNING 2023-04-18 02:46:58,748 basehttp log_message 72312 6190592000 "GET /docs/ HTTP/1.1" 404 2457
INFO 2023-04-18 02:51:02,169 autoreload trigger_reload 72312 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/urls.py changed, reloading.
INFO 2023-04-18 02:51:02,650 autoreload run_with_reloader 72382 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 02:51:38,025 autoreload trigger_reload 72382 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/urls.py changed, reloading.
INFO 2023-04-18 02:51:38,423 autoreload run_with_reloader 72400 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 02:51:57,182 autoreload trigger_reload 72400 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/urls.py changed, reloading.
INFO 2023-04-18 02:51:57,618 autoreload run_with_reloader 72412 8486771520 Watching for file changes with StatReloader
WARNING 2023-04-18 02:52:43,300 log log_response 72412 6130921472 Not Found: /redos/
WARNING 2023-04-18 02:52:43,301 basehttp log_message 72412 6130921472 "GET /redos/ HTTP/1.1" 404 2624
ERROR 2023-04-18 02:52:49,103 log log_response 72412 6130921472 Internal Server Error: /redoc/
Traceback (most recent call last):
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/drf_yasg2/views.py", line 110, in get
    schema = generator.get_schema(request, self.public)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/schemas/generators.py", line 226, in get_schema
    raise NotImplementedError(".get_schema() must be implemented in subclasses.")
NotImplementedError: .get_schema() must be implemented in subclasses.
ERROR 2023-04-18 02:52:49,104 basehttp log_message 72412 6130921472 "GET /redoc/ HTTP/1.1" 500 102185
INFO 2023-04-18 02:53:20,459 basehttp log_message 72412 6130921472 "GET /swagger HTTP/1.1" 301 0
ERROR 2023-04-18 02:53:20,493 log log_response 72412 6147747840 Internal Server Error: /swagger/
Traceback (most recent call last):
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/drf_yasg2/views.py", line 110, in get
    schema = generator.get_schema(request, self.public)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/schemas/generators.py", line 226, in get_schema
    raise NotImplementedError(".get_schema() must be implemented in subclasses.")
NotImplementedError: .get_schema() must be implemented in subclasses.
ERROR 2023-04-18 02:53:20,494 basehttp log_message 72412 6147747840 "GET /swagger/ HTTP/1.1" 500 102221
INFO 2023-04-18 02:55:09,824 autoreload run_with_reloader 72483 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 02:55:27,043 autoreload run_with_reloader 72493 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 02:55:57,646 autoreload run_with_reloader 72506 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:01:34,824 autoreload run_with_reloader 73985 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:01:57,599 autoreload run_with_reloader 73994 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:03:00,785 autoreload run_with_reloader 74011 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:07:21,297 autoreload run_with_reloader 75569 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:09:01,433 autoreload trigger_reload 75569 8486771520 /Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/request.py changed, reloading.
INFO 2023-04-18 03:09:02,126 autoreload run_with_reloader 75700 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:09:14,081 autoreload run_with_reloader 76492 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:10:11,007 autoreload run_with_reloader 76513 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:10:38,537 autoreload run_with_reloader 76527 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:12:41,153 autoreload run_with_reloader 76580 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:12:52,268 autoreload run_with_reloader 76588 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:17:11,976 autoreload run_with_reloader 76650 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:17:37,294 autoreload run_with_reloader 76671 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:18:43,635 autoreload run_with_reloader 79091 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:18:58,234 basehttp log_message 79091 6200700928 "GET /home/demo/ HTTP/1.1" 200 5151
INFO 2023-04-18 03:21:59,523 autoreload trigger_reload 79091 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/settings/dev.py changed, reloading.
INFO 2023-04-18 03:21:59,970 autoreload run_with_reloader 79153 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:22:21,565 autoreload trigger_reload 79153 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/urls.py changed, reloading.
INFO 2023-04-18 03:22:21,956 autoreload run_with_reloader 79170 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:22:39,484 autoreload run_with_reloader 79191 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:24:15,829 autoreload run_with_reloader 79994 8486771520 Watching for file changes with StatReloader
WARNING 2023-04-18 03:24:33,729 log log_response 79994 10754224128 Not Found: /docs/
WARNING 2023-04-18 03:24:33,730 basehttp log_message 79994 10754224128 "GET /docs/ HTTP/1.1" 404 2621
WARNING 2023-04-18 03:24:49,065 log log_response 79994 10754224128 Not Found: /docs、
WARNING 2023-04-18 03:24:49,066 basehttp log_message 79994 10754224128 "GET /docs%E3%80%81 HTTP/1.1" 404 2633
WARNING 2023-04-18 03:25:02,591 log log_response 79994 10754224128 Not Found: /docs/redoc
WARNING 2023-04-18 03:25:02,592 basehttp log_message 79994 10754224128 "GET /docs/redoc HTTP/1.1" 404 2636
INFO 2023-04-18 03:25:13,721 basehttp log_message 79994 10754224128 "GET /redoc HTTP/1.1" 301 0
ERROR 2023-04-18 03:25:13,763 log log_response 79994 10771050496 Internal Server Error: /redoc/
Traceback (most recent call last):
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/drf_yasg2/views.py", line 110, in get
    schema = generator.get_schema(request, self.public)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/schemas/generators.py", line 226, in get_schema
    raise NotImplementedError(".get_schema() must be implemented in subclasses.")
NotImplementedError: .get_schema() must be implemented in subclasses.
ERROR 2023-04-18 03:25:13,764 basehttp log_message 79994 10771050496 "GET /redoc/ HTTP/1.1" 500 102185
INFO 2023-04-18 03:29:05,592 autoreload trigger_reload 79994 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/settings/dev.py changed, reloading.
INFO 2023-04-18 03:29:05,871 autoreload run_with_reloader 80083 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:30:05,393 autoreload run_with_reloader 80117 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:30:23,752 autoreload run_with_reloader 80863 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:31:09,896 autoreload trigger_reload 80863 8486771520 /Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/drf_yasg/errors.py changed, reloading.
INFO 2023-04-18 03:31:10,217 autoreload run_with_reloader 82726 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:31:20,815 autoreload run_with_reloader 83190 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:32:19,535 autoreload trigger_reload 83190 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/settings/dev.py changed, reloading.
INFO 2023-04-18 03:32:19,966 autoreload run_with_reloader 83215 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:34:35,893 autoreload trigger_reload 83215 8486771520 /Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/drf_yasg/inspectors/query.py changed, reloading.
INFO 2023-04-18 03:34:36,636 autoreload run_with_reloader 85169 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:34:45,728 autoreload run_with_reloader 85624 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:34:55,899 basehttp log_message 85624 6160134144 "GET /redoc/ HTTP/1.1" 200 932
INFO 2023-04-18 03:34:55,933 basehttp log_message 85624 6176960512 "GET /static/drf-yasg/insQ.min.js HTTP/1.1" 200 2093
INFO 2023-04-18 03:34:55,933 basehttp log_message 85624 6160134144 "GET /static/drf-yasg/style.css HTTP/1.1" 200 1047
INFO 2023-04-18 03:34:55,933 basehttp log_message 85624 6193786880 "GET /static/drf-yasg/redoc-init.js HTTP/1.1" 200 2566
INFO 2023-04-18 03:34:55,936 basehttp log_message 85624 6210613248 "GET /static/drf-yasg/redoc/redoc.min.js HTTP/1.1" 200 998563
INFO 2023-04-18 03:34:56,036 basehttp log_message 85624 6210613248 "GET /redoc/?format=openapi HTTP/1.1" 200 661
INFO 2023-04-18 03:34:56,112 basehttp log_message 85624 6210613248 "GET /static/drf-yasg/redoc/redoc-logo.png HTTP/1.1" 200 4969
WARNING 2023-04-18 03:35:26,988 log log_response 85624 6210613248 Not Found: /swagger/
WARNING 2023-04-18 03:35:26,988 basehttp log_message 85624 6210613248 "GET /swagger/ HTTP/1.1" 404 2734
WARNING 2023-04-18 03:35:33,567 log log_response 85624 6210613248 Not Found: /swagger/
WARNING 2023-04-18 03:35:33,568 basehttp log_message 85624 6210613248 "GET /swagger/ HTTP/1.1" 404 2734
INFO 2023-04-18 03:35:55,434 autoreload trigger_reload 85624 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/urls.py changed, reloading.
INFO 2023-04-18 03:35:55,866 autoreload run_with_reloader 85647 8486771520 Watching for file changes with StatReloader
WARNING 2023-04-18 03:35:58,058 log log_response 85647 6133428224 Not Found: /doc/
WARNING 2023-04-18 03:35:58,059 basehttp log_message 85647 6133428224 "GET /doc/ HTTP/1.1" 404 2718
ERROR 2023-04-18 03:36:11,746 log log_response 85647 6133428224 Internal Server Error: /docs/
Traceback (most recent call last):
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/schemas/views.py", line 48, in handle_exception
    return super().handle_exception(exc)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/schemas/views.py", line 37, in get
    schema = self.schema_generator.get_schema(request, self.public)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/schemas/coreapi.py", line 156, in get_schema
    links = self.get_links(None if public else request)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/schemas/coreapi.py", line 143, in get_links
    link = view.schema.get_link(path, method, base_url=self.url)
AttributeError: 'AutoSchema' object has no attribute 'get_link'
ERROR 2023-04-18 03:36:11,748 basehttp log_message 85647 6133428224 "GET /docs/ HTTP/1.1" 500 108990
INFO 2023-04-18 03:36:16,770 basehttp log_message 85647 6133428224 "GET /doc HTTP/1.1" 200 2157
INFO 2023-04-18 03:36:16,793 basehttp log_message 85647 6133428224 "GET /static/drf-yasg/style.css HTTP/1.1" 304 0
INFO 2023-04-18 03:36:16,797 basehttp log_message 85647 6133428224 "GET /static/drf-yasg/insQ.min.js HTTP/1.1" 304 0
INFO 2023-04-18 03:36:16,809 basehttp log_message 85647 6217560064 "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1" 200 14964
INFO 2023-04-18 03:36:16,810 basehttp log_message 85647 6200733696 "GET /static/drf-yasg/immutable.min.js HTTP/1.1" 200 56904
INFO 2023-04-18 03:36:16,812 basehttp log_message 85647 6150254592 "GET /static/drf-yasg/swagger-ui-dist/swagger-ui.css HTTP/1.1" 200 143858
INFO 2023-04-18 03:36:16,817 basehttp log_message 85647 6183907328 "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js HTTP/1.1" 200 324275
INFO 2023-04-18 03:36:16,819 basehttp log_message 85647 6167080960 "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js HTTP/1.1" 200 1086481
INFO 2023-04-18 03:36:16,915 basehttp log_message 85647 6167080960 "GET /doc?format=openapi HTTP/1.1" 200 661
INFO 2023-04-18 03:36:16,916 basehttp log_message 85647 6183907328 "GET /static/drf-yasg/swagger-ui-dist/favicon-32x32.png HTTP/1.1" 200 628
INFO 2023-04-18 03:38:08,200 autoreload trigger_reload 85647 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/urls.py changed, reloading.
INFO 2023-04-18 03:38:08,704 autoreload run_with_reloader 85686 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:38:26,943 basehttp log_message 85686 6160134144 "GET /home/demo/ HTTP/1.1" 200 2
ERROR 2023-04-18 03:38:44,577 log log_response 85686 6160134144 Internal Server Error: /docs/
Traceback (most recent call last):
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/schemas/views.py", line 48, in handle_exception
    return super().handle_exception(exc)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/schemas/views.py", line 37, in get
    schema = self.schema_generator.get_schema(request, self.public)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/schemas/coreapi.py", line 156, in get_schema
    links = self.get_links(None if public else request)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/schemas/coreapi.py", line 143, in get_links
    link = view.schema.get_link(path, method, base_url=self.url)
AttributeError: 'AutoSchema' object has no attribute 'get_link'
ERROR 2023-04-18 03:38:44,578 basehttp log_message 85686 6160134144 "GET /docs/ HTTP/1.1" 500 108990
INFO 2023-04-18 03:38:55,668 basehttp log_message 85686 6160134144 "GET /redoc/ HTTP/1.1" 200 932
INFO 2023-04-18 03:38:55,693 basehttp log_message 85686 6160134144 "GET /static/drf-yasg/style.css HTTP/1.1" 304 0
INFO 2023-04-18 03:38:55,695 basehttp log_message 85686 6160134144 "GET /static/drf-yasg/insQ.min.js HTTP/1.1" 304 0
INFO 2023-04-18 03:38:55,695 basehttp log_message 85686 10754224128 "GET /static/drf-yasg/redoc/redoc.min.js HTTP/1.1" 304 0
INFO 2023-04-18 03:38:55,695 basehttp log_message 85686 10771050496 "GET /static/drf-yasg/redoc-init.js HTTP/1.1" 304 0
INFO 2023-04-18 03:38:55,782 basehttp log_message 85686 10771050496 "GET /redoc/?format=openapi HTTP/1.1" 200 661
INFO 2023-04-18 03:40:09,408 autoreload run_with_reloader 85721 8486771520 Watching for file changes with StatReloader
WARNING 2023-04-18 03:40:12,104 log log_response 85721 6171488256 Not Found: /
WARNING 2023-04-18 03:40:12,104 basehttp log_message 85721 6171488256 "GET / HTTP/1.1" 404 2689
INFO 2023-04-18 03:40:22,546 basehttp log_message 85721 6171488256 "GET /docs1 HTTP/1.1" 301 0
ERROR 2023-04-18 03:40:22,586 log log_response 85721 6188314624 Internal Server Error: /docs1/
Traceback (most recent call last):
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/schemas/views.py", line 48, in handle_exception
    return super().handle_exception(exc)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/schemas/views.py", line 37, in get
    schema = self.schema_generator.get_schema(request, self.public)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/schemas/coreapi.py", line 156, in get_schema
    links = self.get_links(None if public else request)
  File "/Users/qiaopengjun/miniconda3/envs/mingde/lib/python3.10/site-packages/rest_framework/schemas/coreapi.py", line 143, in get_links
    link = view.schema.get_link(path, method, base_url=self.url)
AttributeError: 'AutoSchema' object has no attribute 'get_link'
ERROR 2023-04-18 03:40:22,587 basehttp log_message 85721 6188314624 "GET /docs1/ HTTP/1.1" 500 109007
INFO 2023-04-18 03:41:14,214 autoreload run_with_reloader 85763 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:41:19,710 basehttp log_message 85763 6138376192 "GET /docs1/ HTTP/1.1" 200 11779
INFO 2023-04-18 03:41:19,750 basehttp log_message 85763 6222508032 "GET /docs1/schema.js HTTP/1.1" 200 504
INFO 2023-04-18 03:41:19,751 basehttp log_message 85763 6172028928 "GET /static/rest_framework/docs/css/base.css HTTP/1.1" 200 6156
INFO 2023-04-18 03:41:19,754 basehttp log_message 85763 6138376192 "GET /static/rest_framework/css/bootstrap-theme.min.css HTTP/1.1" 200 23411
INFO 2023-04-18 03:41:19,754 basehttp log_message 85763 6155202560 "GET /static/rest_framework/css/font-awesome-4.0.3.css HTTP/1.1" 200 21658
INFO 2023-04-18 03:41:19,755 basehttp log_message 85763 6172028928 "GET /static/rest_framework/docs/js/api.js HTTP/1.1" 200 10391
INFO 2023-04-18 03:41:19,755 basehttp log_message 85763 6222508032 "GET /static/rest_framework/docs/js/jquery.json-view.min.js HTTP/1.1" 200 2700
INFO 2023-04-18 03:41:19,755 basehttp log_message 85763 6188855296 "GET /static/rest_framework/docs/css/jquery.json-view.min.css HTTP/1.1" 200 1307
INFO 2023-04-18 03:41:19,756 basehttp log_message 85763 6205681664 "GET /static/rest_framework/js/coreapi-0.1.1.js HTTP/1.1" 200 157600
INFO 2023-04-18 03:41:19,803 basehttp log_message 85763 6188855296 "GET /static/rest_framework/fonts/fontawesome-webfont.woff?v=4.0.3 HTTP/1.1" 200 44432
INFO 2023-04-18 03:41:19,853 basehttp log_message 85763 6188855296 "GET /static/rest_framework/docs/img/favicon.ico HTTP/1.1" 200 5430
INFO 2023-04-18 03:43:34,028 autoreload trigger_reload 85763 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/urls.py changed, reloading.
INFO 2023-04-18 03:43:34,455 autoreload run_with_reloader 85803 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:45:35,144 autoreload trigger_reload 85803 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/urls.py changed, reloading.
INFO 2023-04-18 03:45:35,604 autoreload run_with_reloader 85843 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:47:03,125 autoreload trigger_reload 85843 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/urls.py changed, reloading.
INFO 2023-04-18 03:47:03,575 autoreload run_with_reloader 85875 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:48:41,638 autoreload trigger_reload 85875 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/urls.py changed, reloading.
INFO 2023-04-18 03:48:42,076 autoreload run_with_reloader 85915 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 03:57:24,062 basehttp log_message 85915 6161231872 "GET /home/demo/ HTTP/1.1" 200 5151
INFO 2023-04-18 04:33:38,096 autoreload trigger_reload 85915 8486771520 /Users/qiaopengjun/Code/mingde/mingdeapi/mingdeapi/settings/dev.py changed, reloading.
INFO 2023-04-18 04:33:38,618 autoreload run_with_reloader 86533 8486771520 Watching for file changes with StatReloader
INFO 2023-04-18 04:46:00,498 basehttp log_message 86533 6199717888 "GET /docs/ HTTP/1.1" 200 11775
INFO 2023-04-18 04:46:00,534 basehttp log_message 86533 6216544256 "GET /static/rest_framework/css/font-awesome-4.0.3.css HTTP/1.1" 304 0
INFO 2023-04-18 04:46:00,536 basehttp log_message 86533 6199717888 "GET /docs/schema.js HTTP/1.1" 200 504
INFO 2023-04-18 04:46:00,536 basehttp log_message 86533 6233370624 "GET /static/rest_framework/docs/css/jquery.json-view.min.css HTTP/1.1" 304 0
INFO 2023-04-18 04:46:00,537 basehttp log_message 86533 6250196992 "GET /static/rest_framework/docs/css/base.css HTTP/1.1" 304 0
INFO 2023-04-18 04:46:00,537 basehttp log_message 86533 6267023360 "GET /static/rest_framework/js/coreapi-0.1.1.js HTTP/1.1" 304 0
INFO 2023-04-18 04:46:00,538 basehttp log_message 86533 6283849728 "GET /static/rest_framework/css/bootstrap-theme.min.css HTTP/1.1" 304 0
INFO 2023-04-18 04:46:00,538 basehttp log_message 86533 6216544256 "GET /static/rest_framework/docs/js/jquery.json-view.min.js HTTP/1.1" 304 0
INFO 2023-04-18 04:46:00,538 basehttp log_message 86533 6199717888 "GET /static/rest_framework/docs/js/api.js HTTP/1.1" 304 0
INFO 2023-04-18 04:46:00,632 basehttp log_message 86533 6199717888 "GET /static/rest_framework/fonts/fontawesome-webfont.woff?v=4.0.3 HTTP/1.1" 304 0
INFO 2023-04-18 04:47:21,977 basehttp log_message 86533 6199717888 "GET /doc HTTP/1.1" 200 2157
INFO 2023-04-18 04:47:22,013 basehttp log_message 86533 6199717888 "GET /static/drf-yasg/style.css HTTP/1.1" 304 0
INFO 2023-04-18 04:47:22,016 basehttp log_message 86533 6216544256 "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js HTTP/1.1" 304 0
INFO 2023-04-18 04:47:22,016 basehttp log_message 86533 6233370624 "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1" 304 0
INFO 2023-04-18 04:47:22,017 basehttp log_message 86533 6267023360 "GET /static/drf-yasg/swagger-ui-dist/swagger-ui.css HTTP/1.1" 304 0
INFO 2023-04-18 04:47:22,017 basehttp log_message 86533 6283849728 "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js HTTP/1.1" 304 0
INFO 2023-04-18 04:47:22,017 basehttp log_message 86533 6199717888 "GET /static/drf-yasg/immutable.min.js HTTP/1.1" 304 0
INFO 2023-04-18 04:47:22,018 basehttp log_message 86533 6250196992 "GET /static/drf-yasg/insQ.min.js HTTP/1.1" 304 0
INFO 2023-04-18 04:47:22,146 basehttp log_message 86533 6250196992 "GET /doc?format=openapi HTTP/1.1" 200 698
INFO 2023-04-18 04:48:16,309 basehttp log_message 86533 6250196992 "GET /redoc/ HTTP/1.1" 200 932
INFO 2023-04-18 04:48:16,337 basehttp log_message 86533 6250196992 "GET /static/drf-yasg/redoc-init.js HTTP/1.1" 304 0
INFO 2023-04-18 04:48:16,338 basehttp log_message 86533 6199717888 "GET /static/drf-yasg/redoc/redoc.min.js HTTP/1.1" 304 0
INFO 2023-04-18 04:48:16,473 basehttp log_message 86533 6199717888 "GET /redoc/?format=openapi HTTP/1.1" 200 698
INFO 2023-04-18 04:49:57,289 basehttp log_message 86533 6199717888 "GET /redoc/ HTTP/1.1" 200 932
INFO 2023-04-18 04:49:57,414 basehttp log_message 86533 6199717888 "GET /redoc/?format=openapi HTTP/1.1" 200 698
